### ISSUE TITLE
Supporting semantic elements

### ISSUE THREAD

**Issue by qkreltms** [llorentegerman/simple-flexbox#4]
**How about supporting semantic elements?**

I think all we need to change is Layout's element ```div``` to semantic elements.
```js
        return (<div style={layoutStyles} {...ownProps}>
            {this.props.children}
        </div>);
```

I have example link : https://github.com/nachoaIvarez/flexbox-react/blob/master/src/Flexbox.jsx

**Comment by llorentegerman** [comment]
Yes, no problem, I will add something like this in the next version, hopefully in a few days

**Comment by llorentegerman** [comment]
Included in version **2.1.0** 
Usage example:
```
<Column>
    <Row element="article">
        <span>My parent is an article</span>
    </Row>
    <Row>
        <span>My parent is a div</span>
    </Row>
    <Row element="section">
        <span>My parent is a section</span>
    </Row>
</Column>
```
